src/ff/readme.md

```diff
@@ -0,0 +1,9 @@
+# Copyright
+
+```
+    Copyright 2018 0kims association.
+
+    This files in this folder part of ffjavascript library rewritten by Typescript.
```

Comment:
"Files in this folder" without first "this".
"Rewritten *to* Typescript".
And does it make sense to contribute back to ffjavascript?
On the long run it would be more time consuming to maintain both versions.